File: agenter/api/src/main/java/me/jiahuan/stf/agenter/pojo/DeviceInfo.java
package me.jiahuan.stf.agenter.pojo;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class DeviceInfo {
    private String deviceId;
    private String manufacturer;
    private String model;
    private String os;
    private String osVersion;
    private Size size;
}

File: device/app/src/main/java/me/jiahuan/stf/device/server/device/DeviceInfo.kt
package me.jiahuan.stf.device.server.device

import me.jiahuan.stf.device.server.model.Size

/**
 * 设备信息
 */
class DeviceInfo(
    var manufacturer: String,
    var model: String,
    var os: String,
    var osVersion: String,
    var size: Size
)